config.yaml

```diff
@@ -618,6 +644,27 @@ guilds:
       - name: "TOP Cleared"
         type: "Cleared"
         color: 0x6534c6
+      - name: "Reclear TOP"
+        type: "Reclear"
+        color: 0x6534c6
+      - name: "The Alpha Color"
+        type: "Name Color"
+        color: 0x6534c6
+  # - ids: []
```

Comment:
Prepared for FRU release, can be uncommented when the fight goes live.

Reply:
just like the very first PR of the chain, let's add a `// TODO: ... ` comment